### DISCUSSION TITLE
java的一些问题

### DISCUSSION THREAD

**Discussion by Pangzongawa** [Meloong-Git/PCL#1910] [提问] [answered]
下载了jvav20后，手动导入到pcl2中，后来在文件夹中删除jvav20，pcl2依旧显示有这个jvav20的路径，也能启动游戏，不清楚咋回事先来这里提问一下

**Comment by WForst-Breeze** [comment, marked as answer]
可能你在删除该 Java 后没有重启 PCL2 或没有重新搜索 Java，导致 PCL2 依然显示该 Java。而启动游戏时 PCL2 自动选择了其他合适的 Java 使得游戏成功启动
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/97ee8889-3fd7-40f0-9b1e-c2b3170e03b4)

**Comment by Pangzongawa** [comment]
重启的话并不能解决，重新搜索后倒是解决了问题
